solr/core/src/test/org/apache/solr/cli/TestSolrCLIRunExample.java

```diff
@@ -325,6 +330,99 @@ public void testFilmsExample() throws Exception {
     testExample("films");
   }
 
+  @Test
+  public void testAdditionalArgs() throws Exception {
+    String testStandaloneMode = LuceneTestCase.rarely() ? "--user-managed" : "";
+    Path defaultSolrHomeDir = ExternalPaths.SERVER_HOME;
+    if (!Files.isDirectory(defaultSolrHomeDir)) {
```

Comment:
I don't know if this is required...    I believe that all our tests that use `ExternalPaths.SERVER_HOME` assume it's valid..   Altenratively, I think there is a assert or assume type operation that only runs the test when a criteria is met for things that ARE variable!